kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/kview.css

```diff
@@ -4092,3 +4092,117 @@ Styling a context menu for kview ui/ux controls
     -fx-background-insets: 0;
     -fx-background-radius: 6;
 }
+
+/*****************************
+* next gen reasoner styles
+*****************************/
+.grey1-label {
+    -fx-text-fill: -Grey-1;
+    -fx-font-family: "Noto Sans";
+    -fx-font-size: 14px;    
+}
+
+.grey1-x {
+    -height: 18px;
+    -width: 18px;
+    -fx-min-height: -height;
+    -fx-pref-height: -height;
+    -fx-max-height: -height;
+    -fx-content-display: graphic-only;
+    -fx-min-width: -width;
+    -fx-pref-width: -width;
+    -fx-max-width: -width;
+    -fx-background-color: -Grey-1;
+    -fx-shape: "M19 6.41L17.59 5L12 10.59L6.41 5L5 6.41L10.59 12L5 17.59L6.41 19L12 13.41L17.59 19L19 17.59L13.41 12L19 6.41Z";
+}
+
+.classification-date-titled-pane {
```

Comment:
Can the following style classes be more generic or reusable?
Maybe like the following: (i'm not sure where in the UI this is applied. Hopefully you get the idea) Some JavaFX have implicit style class such as a `.title-pane`, so adding the control can be redundant sounding.
```css
.titled-pane.classification-date {

}
```

Reply:
fixed